app/controllers/course/statistics/aggregate_controller.rb

```diff
@@ -34,6 +34,13 @@ def all_assessments
     fetch_all_assessment_related_statistics_hash
   end
 
+  def download_score_summary
```

Comment:
Should to put `authorize!` to restrict, otherwise non-elevated roles would be able to download the scores.

Reply:
the authorisation of this function is already governed inside `class Course::Statistics::Controller`, which is the parent class of this controller